OnlySwitch.xcodeproj/project.pbxproj

```diff
@@ -689,6 +689,8 @@
 		1081EF19287F28C5005849B5 /* so */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = so; path = so.lproj/Intents.strings; sourceTree = "<group>"; };
 		1081EF1A287F28C5005849B5 /* so */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = so; path = so.lproj/Localizable.strings; sourceTree = "<group>"; };
 		29107C0828111D3A00E88A86 /* ja */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = ja; path = ja.lproj/Localizable.strings; sourceTree = "<group>"; };
+		40109DE32B6B18E0005D4802 /* pt-BR */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = "pt-BR"; path = "pt-BR.lproj/Intents.strings"; sourceTree = "<group>"; };
```

Comment:
When you remove the Intents localization file, this line should be also removed.